spark3/src/main/java/org/apache/iceberg/spark/FileRewriteCoordinator.java

```diff
@@ -93,7 +96,7 @@ public void commitRewrite(Table table, Set<String> fileSetIDs) {
     return Collections.unmodifiableSet(rewrittenDataFiles);
   }
 
-  private Set<DataFile> fetchNewDataFiles(Table table, Set<String> fileSetIDs) {
+  public Set<DataFile> fetchNewDataFiles(Table table, Set<String> fileSetIDs) {
```

Comment:
I think this can now just accept a single file set id.

Reply:
I have to delete code out of FileRewriteCoordinator to do this since the function is used in CommitOrRewrite. I can do this in the followup pr?